set_1_basics/challenge_6_break_repeating_key_xor/decode_base64.cpp

```diff
@@ -33,26 +33,22 @@ std::istream & operator>>(std::istream & input,
         input >> base64Digit;
         if (!input) input.setstate(std::ios_base::failbit);
         manipulator.d_output[0] = decode_base64::convertFromBase64(base64Digit)
-                                  << (
-                                        decode_base64::s_numBitsInOctet
-                                        - decode_base64::s_numBitsInBase64Digit
-                                     );
+                                  << (decode_base64::s_numBitsInOctet
+                                      - decode_base64::s_numBitsInBase64Digit);
 
         input >> base64Digit;
         if (!input) input.setstate(std::ios_base::failbit);
         uint8_t value = decode_base64::convertFromBase64(base64Digit);
         static const size_t s_numRemainingBitsInFirstOctet =
-                decode_base64::s_numBitsInOctet
-                - decode_base64::s_numBitsInBase64Digit;
+            decode_base64::s_numBitsInOctet
+            - decode_base64::s_numBitsInBase64Digit;
         static const size_t s_numBitsInSecondOctet =
-                decode_base64::s_numBitsInBase64Digit
-                - s_numRemainingBitsInFirstOctet;
+            decode_base64::s_numBitsInBase64Digit
+            - s_numRemainingBitsInFirstOctet;
         manipulator.d_output[0].get() += value >> s_numBitsInSecondOctet;
-        manipulator.d_output[1] = (value & ((1u << s_numBitsInSecondOctet) - 1))
-                                  << (
-                                        decode_base64::s_numBitsInOctet
-                                        - s_numBitsInSecondOctet
-                                     );
+        manipulator.d_output[1] =
+            (value & ((1u << s_numBitsInSecondOctet) - 1))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Uppercase the literal suffix. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3ALiteralSuffix)